src/QueuedWriter.php

```diff
@@ -130,8 +131,13 @@ private function exportQuery(
     ) {
         $query = $export->query();
 
+        if ($export instanceof WithCount) {
+            $spins = ceil($export->count() / $this->chunkSize);
```

Comment:
You can put the count in a variable and then reuse the spins part. (Avoiding code duplication)